### ISSUE TITLE
Source of data

### ISSUE THREAD

**Issue by CAROLZXYZXY** [StevenWD/HR-BiLSTM#2]
Hi Steven,

Thanks for implementing this paper and making it open. I've got a question. Where does the files data/WebQSP.RE.train(test).with_boundary.withpool.dlnlp.txt comes from. More specifically, what are the pre-preprocess step from the natural language questions. I noticed that the negative relations are only a part of the whole relations, then how to select them?

Thank you very much!

**Comment by StevenWD** [comment]
Hi,


Thank you for your star on github. The training and test data are provided by another github repository: https://github.com/LayneIns/KBQA. I directly use their data in the process of implementation. 


Best wishes,


Yongjian You



At 2018-08-08 04:41:56, "CAROL(XINYI) ZHENG" <notifications@github.com> wrote:


Hi Steven,

Thanks for implementing this paper and making it open. I've got a question. Where does the files data/WebQSP.RE.train(test).with_boundary.withpool.dlnlp.txt comes from. More specifically, what are the pre-preprocess step from the natural language questions. I noticed that the negative relations are only a part of the whole relations, then how to select them?

Thank you very much!
